### ISSUE TITLE
Command-click opens tab at far right of window, not adjacent

### ISSUE THREAD

**Issue by psmalone** [brave/browser-laptop#6907]
- Did you search for similar issues before submitting this one?

Yes

- Describe the issue you encountered:

I want to open a link in a new tab. Doing so from the context menu opens the tab immediately to the right, as expected, even if there are other tabs to the right. Doing so with the command-click keyboard shortcut opens the new tab at the far right of the window.

- Platform (Win7, 8, 10? macOS? Linux distro?):

OS X El Capitan 10.11.6

- Brave Version (revision SHA):

0.13.0 (8ca0127)

- Steps to reproduce:
    1. Open multiple tabs in a single window
    2. Focus on tab in the middle of the window
    3. Command-click a link

- Actual result:
New tab from link opens at far right of window.

- Expected result:
New tab from link opens immediately to the right of the current tab.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Unknown

- Is this an issue in the currently released version?
Yes. New issue in 0.13.0.

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by jhreis** [comment]
This is especially confusing when the furtherest right tab space is in a different tab group than the active tab, since there is visual feedback/response.

**Comment by bradleyrichter** [comment]
@bsclifton Gotta get this one in soon...

**Comment by NejcZdovc** [comment]
I think this will be fixed with version 0.13.2 with this PR #6573

**Comment by bsclifton** [comment]
@NejcZdovc can you confirm this is fixed in master? If so, let's close it out 😄

**Comment by NejcZdovc** [comment]
Sorry my bad this PR #6573 is for close not for open. I can create a similar PR for open as I did for close

**Comment by NejcZdovc** [comment]
I think this is duplication of #2000. @bradleyrichter should there be an option to chose from (next to the current or at the end) or should we always open next to it?

**Comment by bradleyrichter** [comment]
@NejcZdovc Originally this was a new tab settings options in prefs:

![image](https://cloud.githubusercontent.com/assets/13509546/22442794/5f5f0a9e-e6f1-11e6-8351-eb26eb8a6d34.png)

**Comment by NejcZdovc** [comment]
I remember now, #2137  is also related to this issue. So options "To the right" and "At the end" should be added into tabs section. Right?

One more thing. Right now we have everything in one column, but I see that you have multiple columns in multiple sketches. How should we implement this one. This is a current tab layout (latest master):

![image](https://cloud.githubusercontent.com/assets/9574457/22442935/571b55bc-e73d-11e6-921e-48918a182db5.png)

**Comment by bradleyrichter** [comment]
Not sure where this came from? We need to make sure the default matches Chrome.

![image](https://cloud.githubusercontent.com/assets/13509546/22443064/69b948fa-e6f2-11e6-9558-782261a1837b.png)



I'll make a new mockup for a vertical layout.

**Comment by NejcZdovc** [comment]
This comes from #6553

**Comment by bradleyrichter** [comment]
new layout:

![image](https://cloud.githubusercontent.com/assets/13509546/22488822/0250db24-e7c9-11e6-8d1f-976a411e65a6.png)

**Comment by bbondy** [comment]
Is this preference normal @bradleyrichter ? 
I think it's always known when it should be opened directly to the right, when it is related to tab, and when it is not, when it is related to top level UI.   I think we should just match chrome on this like we always have with no UI.

**Comment by bbondy** [comment]
I guess I'm ok with a preference but when you click a top level global UI thing, it should always open all the way to the right, and when it is within a tab it should use that setting? The wording on the pref name is a bit confusing.

**Comment by bbondy** [comment]
needs more thought for wording / the difference between global UI creation of tabs and within tab creation of tabs.  I added a smaller task to 0.13.2 here https://github.com/brave/browser-laptop/issues/6971 @NejcZdovc in case you can fix the regression without the setting for 0.13.2.

**Comment by bradleyrichter** [comment]
@bbondy The main change/improvement is to open a new tab to the right of the current tab when using command+T.

Perhaps we could use a slider (checkbox) and the string:

[ x ]  Always open new tabs to the right of the current tab.
 

Thoughts?

**Comment by NejcZdovc** [comment]
I think that the main question here is if users's would even like to have this option to always open tab at the end? I personally don't need it.

 Because the core problem for this issue is really #6971, which @bbondy opened. When this will be fixed everything should be back to normal (links opened from inside of the page will be loaded next to the current tab, with all possible shortcuts)

**Comment by bradleyrichter** [comment]
@NejcZdovc It is true that most of the pain comes from our bugs causing it to behave differently than Chrome. 

I am fine if we want to shelve this feature and wait for more feedback/requests.

**Comment by NejcZdovc** [comment]
I think that after #6971 will be fixed behavior should be like @bbondy said [above](https://github.com/brave/browser-laptop/issues/6907#issuecomment-276569717), with the current code base.

**Comment by bbondy** [comment]
agreed, sorry for not chiming in earlier

**Comment by bradleyrichter** [comment]
I updated the spec image above.
